nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -23,6 +23,23 @@ in
           If left at the default value, $HOME/.i3/config will be used.
         '';
       };
+
+      debug = mkOption {
+        default = false;
+        example = true;
+        type = types.bool;
+        description = "Enable debug/verbose logging (see -V option)";
+      };
+
+      logFile = mkOption {
+        default = null;
+        example = "$HOME/.i3/i3log";
```

Comment:
Not in this context, as the code defined here is used in the xsession script (at https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/x11/display-managers/default.nix#L148). In that script, $HOME is already set to the home of the right user. That script also loads `~/.Xresources` for example.